src/Framework/PaymentGateways/PaymentGatewayRegister.php

```diff
@@ -9,20 +9,31 @@
 use Give\Framework\PaymentGateways\Exceptions\OverflowException;
 
 /**
+ * @unreleased add deprecated gateways
  * @since 2.18.0
  */
 class PaymentGatewayRegister extends PaymentGatewaysIterator
 {
+    /**
+     * @var string[]
+     */
     protected $gateways = [];
 
     /**
      * Get Gateways
```

Comment:
Please update the description to explain how to use the `$apiVersion` parameter.